main.yml

```diff
@@ -10,11 +10,12 @@ approval:
     reload: catalog/approval
 
 api:
```

Comment:
I know I approved this ID earlier, but I've been thinking on it. `api` is really vague and is confusing to see in this list of apps. Can we change the ID to something like `api-docs` instead?

Reply:
Yeah the api felt a bit fishy for me as well. I'll change it to api docs, good point.